e2e/test-runner/runner_test.go

```diff
@@ -16,20 +29,134 @@ func TestBasicSetup(t *testing.T) {
 	}
 
 	// After completing the setup, the OOBE must have created a new user and the distro launcher must have set it as the default.
-	outputStr := tester.AssertWslCommand("whoami")
-	notRoot := strings.Fields(outputStr)[0]
-	if notRoot == "root" {
-		tester.Logf("%s", outputStr)
-		tester.Fatal("Default user should not be root.")
+	{

```

Comment:
As you realized those blocks could have their own scope, I think we should actually put names on them and call them as functions. Maybe we can reuse some of those functions in other test scenarios beyond the basic setup.

Reply:
Fair enough. Done.